admin/config.yml

```diff
@@ -0,0 +1,46 @@
+backend:
+  name: git-gateway
+  branch: master # Branch to update (optional; defaults to master)
+  site_domain: chihacknight.netlify.com
+
```

Comment:
There is also an editorial workflow we can optionally enable to approve drafts of posts/content